src/PHPUnit/PHPMatcherConstraint.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Coduo\PHPMatcher\PHPUnit;
+
+use Coduo\PHPMatcher\Factory\SimpleFactory;
+use Coduo\PHPMatcher\Matcher;
+
+final class PHPMatcherConstraint extends \PHPUnit_Framework_Constraint
```

Comment:
I'm not sure if this should be marked as final. Seems to be more abstract class than final